flocx_market/api/app.py

```diff
@@ -16,6 +16,6 @@ def create_app(app_name):
     )
     app.config['PROPAGATE_EXCEPTIONS'] = CONF.flask.PROPAGATE_EXCEPTIONS
     api = Api(app)
-    api.add_resource(Offer, '/offer', '/offers/<string:marketplace_offer_id>')
+    api.add_resource(Offer, '/offer', '/offer/<string:marketplace_offer_id>')
```

Comment:
For some reason I didn't push this, this line should be like the line below:
api.add_resource(Offer, '/offer', '/offer/', '/offer/<string:marketplace_offer_id>')

Reply:
That's already corrected in #64.

Reply:
Thanks; I've updated the PR.